data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,224 @@ mission "Remnant: Visit to Paradise 1"
+			`	Her gestures take on an appreciative tone. "The Maudrakoi made extensive use of the materials that make our ships and architecture so distinctive, and this has allowed their cities and technologies to survive the eons since they went extinct. One technology of note is they had an interesting ability to manipulate space. They used this to make large, highly complex objects, and then, for lack of a better term, partially embed them in pocket dimensions so as to not be entirely present. They used this frequently to allow equipment and materials to be much smaller than normal." She shrugs, and gestures with a frustrated tone. "We cannot replicate it, but we have figured out how to disrupt the compression, and thus recover the artifact."`
+			`	The hologram shifts back to showing the Shattered Light. "This ship does not show the same signatures as the Maudrakoi artifacts, but we think it is sufficiently similar that our equipment should work on it too. Hopefully. In any case, though, time is passing, so we should start loading." With that, the Remnant gather their equipment and head out.`
+				accept
+	on complete
+		conversation
+			`With the Shattered Light careening towards the surface of Far Monad, you push the <ship> into a dive. As you focus on finding a safe place to land with a view of the approaching ship, you notice out the corner of your eye that the Remnant team have all suited up and are pushing their equipment to the cargo hatch. A message appears on your console.`
+			`	"Ready for drop, Captain. Just get us along side the Shattered Light without too much forward velocity, we can handle the rest."`
+			`	Moments later your ship pings a point on the ship indicating a gaping hole on the ship. Nodding your assent, the <ship> swoops to a hold position just above it. A glance at your cargo hold display shows the Remnant vanishing out the back lock.  Your exterior cameras show the equipment falling into it on repulsor pallets while the Remnant drop alongside it. A final message appears on your console.`
+			`	"Drop success. Get clear of the arrival zone and pick us up after the dust settles."`
+			`	The <ship> veers and accelerates away from the approaching wreck without waiting for your approval and retreats to a safe distance.`
+
+
+
+mission "Remnant: Shattered Light 3b"
+	name "Remnant: Ghost Touch"
+	description "Help the Remnant experiment on the Shattered Light."
+	source "Viminal"
+	destination "Viminal"
+	waypoint "Coluber"
+	to offer
+		has "Remnant: Shattered Light 2: done"
+	to complete
+		has "Remnant: Shattered Light 3: done"
+
+
+
+mission "Remnant: Shattered Light 4"
```

Comment:
I ran out of time. This is the last part I reviewed at. 

_Don't you dare remove this till I start a new review chain._